app/code/Magento/Wishlist/Controller/Index/DownloadCustomOption.php

```diff
@@ -98,8 +114,8 @@ public function execute()
             if ($secretKey == $info['secret_key']) {
                 $this->_fileResponseFactory->create(
                     $info['title'],
-                    ['value' => $info['quote_path'], 'type' => 'filename'],
-                    DirectoryList::ROOT,
+                    ['value' => $this->_rootDir->getRelativePath($info['quote_path']), 'type' => 'filename'],
+                    $this->rootDirBasePath,
```

Comment:
Hi @rani-webkul,
Wouldn't it be enough to chnage only `DirectoryList::ROOT` to `DirectoryList::MEDIA`? Looks like that was the main issue why it returned no route page.

Reply:
done..

Reply:
Still it works after this change? :)

Reply:
yes.. it's working.. I have verified it.

Reply:
thanks.. :+1: